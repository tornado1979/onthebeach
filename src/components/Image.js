import React from 'react';
import PropTypes from 'prop-types';
import defaultImage from '../images/hotel-image-1.png';

const Image = (props) => {
  return (
    <div>
      <img src={props.imgUrl} />
    </div>
  )
}

Image.defaultProps = {
  imgUrl: defaultImage
}
Image.propTypes = {
  imgUrl: PropTypes.string
};

export default Image;
