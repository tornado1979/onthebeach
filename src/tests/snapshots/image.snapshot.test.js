import React from 'react';
import renderer from 'react-test-renderer';
import Image from '../../components/Image';

describe('Image', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Image/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
