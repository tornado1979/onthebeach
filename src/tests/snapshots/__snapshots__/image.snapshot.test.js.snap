// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders correctly 1`] = `
<div>
  <img
    src="hotel-image-1.png"
    style={
      Object {
        "width": "100%",
      }
    }
  />
</div>
`;
